README.md

```diff
@@ -11,13 +11,18 @@ This repo is tied to the [associated Docker image](https://hub.docker.com/r/karu
 
 These items go in the `source` fields of the resource type. Bold items are required:
 
- * **`bitbucket_url`** - *base* URL of the bitbucket instance, including a trailing slash. (example: `https://bitbucket.mynetwork.com/`)
  * **`bitbucket_user`** - Login username of someone with rights to the repository being updated.
- * **`bitbucket_password`** - Password for the mentioned user
+ * **`bitbucket_password`** - Password for the mentioned user. For Bitbucket Cloud this will need to be an [app password](https://confluence.atlassian.com/bitbucket/app-passwords-828781300.html)
```

Comment:
Sorry I accidently deleted my comments. Yes only read and write permissions for the repository are required. I've updated the documentation.